wheatley/row_generation/helpers.py

```diff
@@ -74,3 +74,21 @@ def convert_to_bell_string(bell: int) -> str:
 def rounds(number_of_bells: int) -> Row:
     """ Generate rounds on the given number of bells. """
     return Row([Bell.from_number(i) for i in range(1, number_of_bells + 1)])
+
+
+def generate_starting_row(number_of_bells: int, start_row_string: str = None) -> Row:
+    """ Generate the starting row as rounds or a custom input. """
+    if start_row_string is None:
+        return rounds(number_of_bells)
+
+    start_row = [Bell.from_str(i) for i in start_row_string]
+    if len(start_row) > len(set(start_row)):
+        raise ValueError(f"starting row '{start_row_string}' contains the same bell multiple times")
```

Comment:
In a similar vein to the other error message, I think this should say actually what bell is duplicated (since that's really helpful for the user).

Reply:
Yeah, i thought about that, was just trying to avoid having too many loops but I'll add it in :)

Reply:
I think it's clear enough (the string ought to be at most 16 characters), without needing the logic of which bell it is

> starting row '5321246' contains the same bell multiple times